src/components/Header/HeaderMenu.tsx

```diff
@@ -1,21 +1,26 @@
+import { useConfigurationStore } from "@/stores/configurationStore";
 import Link from "next/link";
 import { usePathname } from "next/navigation";
-import { useConfigurationStore } from "@/stores/configurationStore";
 
 export default function HeaderMenu() {
     const pathname = usePathname();
     const { menu } = useConfigurationStore();
-    
-    return (<section>
-        {menu.map(element => 
-            <Link
-                className={`px-2 text-lg text-primary ${pathname === element.pathName ? "font-bold text-success" : ""}`}
-                href={element.pathName}
-                key={element.text}
-            >
-                {element.text}
-            </Link>
-        )}
-    </section>
+
+    return (
+        <section className="flex flex-col">
+            <div className="mt-6 flex items-center gap-4 lg:gap-0">
```

Comment:
Probably leftover after the removal of the switch?

Reply:
yep, removed